completion/handlers.go

```diff
@@ -0,0 +1,75 @@
+package completion
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/coder/serpent"
+)
+
+// FileHandler returns a handler that completes files, using the
+// given filter func, which may be nil.
+func FileHandler(filter func(info os.FileInfo) bool) serpent.CompletionHandlerFunc {
+	return func(inv *serpent.Invocation) []string {
+		return ListFiles(inv.CurWord, filter)
+	}
+}
+
+func FileListHandler(filter func(info os.FileInfo) bool) serpent.CompletionHandlerFunc {
```

Comment:
It's unclear from the signature how this differs from `FileHandler`. Either rework the signature or add a comment.